ansys/mapdl/core/plotting.py

```diff
@@ -193,6 +230,9 @@ def general_plotter(meshes, points, labels,
     if title:
         pl.add_title(title)
 
+    if return_plotter:
+        return pl
```

Comment:
This prevents savfig - so either mention in doc-string for parameter savfig, or move `if return_plotter` below `if savefig`

Reply:
Good point.  I've added additional branching to permit both `save_fig` and `return_plotter`.